website_sale_checkout_skip_payment/README.rst

```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+==================================
+Website Sale Checkout Skip Payment
+==================================
+
```

Comment:
This module wants to cover the situation that you current customers could make their own orders maintaining their payment term and payment mode and not paying with a bank transfer, PayPal or similar.